content/en/blog/2024/fuzzing-audit-results.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Opentelemetry Collector Completes Fuzzing Audit
+linkTitle: Fuzzing Audit Results
+date: 2024-12-19
```

Comment:
@AdamKorcz @open-telemetry/docs-approvers do we want to publish this today or move it early next year?

Reply:
I say we publish this week.